### PR TITLE
Add CI linting w/ GitHub Actions

### PR DESCRIPTION
**Pull request by nikosavola**
## Description of change

Adds Continuous Integration linting using GitHub Actions. This should help development by making it easy to notice non-standard code automatically in PRs.

## How has this been tested?

Tested in my fork[ to fail with non-linted code](https://github.com/nikosavola/strava2kilometrikisa/actions/runs/1081233382) and to [succeed with linted code](https://github.com/nikosavola/strava2kilometrikisa/actions/runs/1081223708).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```
Comment (hugovk) [inline .github/workflows/ci.yml:6]:
```suggestion
```

Maybe remove this branch restriction? That way contributors can test their feature branches *before* creating potentially failing PRs.

Comment (nikosavola) [reply .github/workflows/ci.yml:6]: True. I think in this case the cleanest syntax becomes
```yaml
on: [push, pull_request]
```
I'll commit this

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (onefastsnail) [inline .github/workflows/ci.yml:11]: This is a reminder to myself to add a `.nvm` file to use to fetch the Node version and use here to keep versions aligned. And alongside easing development and deployments 😅 .

**Review by onefastsnail — APPROVED** [review]
Awesome, thank you! 🙏

**Pull request merged**